### PR TITLE
CAMEL-14303: Handle duplicate matchOnUriPrefix on api docs.

### PR DESCRIPTION
**Pull request by bobpaulin**

### CODE REVIEW

File: components/camel-netty-http/src/main/java/org/apache/camel/component/netty/http/NettyHttpComponent.java [hunk 1]
```diff
@@ -393,15 +393,15 @@ Consumer doCreateConsumer(CamelContext camelContext, Processor processor, String
 
         // allow HTTP Options as we want to handle CORS in rest-dsl
         boolean cors = config.isEnableCORS();
+        
+        if(api) {
```
Comment (davsclaus) [inline components/camel-netty-http/src/main/java/org/apache/camel/component/netty/http/NettyHttpComponent.java]: Thanks can you fix the checkstyle issue, eg space after if

**Comment by davsclaus** [comment]
Can you do this for the other http components that also support rest api, like jetty, undertow etc. And do PRs for them

**Comment by oscerd** [comment]
I'll merge this for the moment :-) Thanks @bobpaulin

**Comment by bobpaulin** [comment]
> 
> 
> Can you do this for the other http components that also support rest api, like jetty, undertow etc. And do PRs for them

Sure will take a look.  This logic looks very similar in these components.  Perhaps something that can be unified a bit in camel-util?

**Comment by oscerd** [comment]
Merged on master and 3.0.x, I'll leave the issue open for the other components

**Comment by oscerd** [comment]
Thanks @bobpaulin

**Pull request closed without merging**